### PR TITLE
ensure all ids are unique

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Sometimes duplicate ids slip through. lets change them to enforce uniqueness

# Needed By (Date)
- whenever

# Urgency
- low

# Steps to Test

1. Checkout this branch and clear cache
2. ensure no duplicate ids

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: stanford_basic.theme [hunk 1]
```diff
@@ -8,6 +8,17 @@
 use Drupal\Core\Cache\CacheableMetadata;
 use Drupal\Component\Utility\Html;
 
+/**
+ * Implements hook_preprocess().
+ */
+function stanford_basic_preprocess(&$vars, $hook) {
+  // Verify ALL id attributes are unique.
+  if (!empty($vars['attributes']['id'])) {
+    $vars['attributes']['id'] = Html::getUniqueId($vars['attributes']['id']);
```
Comment (sherakama) [inline stanford_basic.theme]: This seems super aggressive. 

Can you add an example scenario where I can reproduce this?

Comment (pookmish) [reply stanford_basic.theme]: Currently no https://archaeology-prod.stanford.edu/people/department-faculty there are two divs with the id `edit-actions`

Comment (pookmish) [reply stanford_basic.theme]: this functionality doens't change ID's unless they already exist on the page.

**Comment by sherakama** [comment]
@pookmish Have you been running this anywhere for some time? I am wondering if any new quirks have popped up.

**Comment by pookmish** [comment]
yep https://github.com/SU-HSDO/suhumsci/blob/develop/docroot/themes/humsci/su_humsci_theme/su_humsci_theme.theme#L133

**Comment by sherakama** [comment]
Ok, if anything, this is the belt & suspenders approach so let's give it a whirl.

**Pull request merged**